### PR TITLE
Added sensor management to the docs

### PR DESCRIPTION
**Pull request by qstorey**

### CODE REVIEW

File: docs/source/dashboard/configuration/sensors_page.rst [hunk 1]
```diff
@@ -11,11 +11,58 @@ The sensors tab consists of 2 sections:
...
-TODO
+The Sensor Management tab allows you to view and manage all of your sensors. In the list you get a quick overview of the each sensor, their group, configured SSIDs and current IP address.
+
+.. image:: /_static/screenshots/sensor_management_list.png
+
+You can bring up more details on a sensor by click on the corresponding row in the list.
+
+.. image:: /_static/screenshots/sensor_detail.png
+
+To edit a sensors configuration, click on the edit button.
```
Comment (mchampanis) [inline docs/source/dashboard/configuration/sensors_page.rst]: text -> `Edit` and make it bold? and so on, for other buttons/links mentioned

File: docs/source/dashboard/configuration/sensors_page.rst [hunk 2]
```diff
@@ -11,11 +11,58 @@ The sensors tab consists of 2 sections:
...
+
+General
+-------
+
+- Name: This gives the sensor a friendly name on the dashboard.
```
Comment (rsdouglas) [inline docs/source/dashboard/configuration/sensors_page.rst]: >Name: This gives the sensor a friendly name on the dashboard, usually referring to a location.

File: docs/source/dashboard/configuration/sensors_page.rst [hunk 3]
```diff
@@ -11,11 +11,58 @@ The sensors tab consists of 2 sections:
...
+- Notes: You can create arbitrary notes about your sensor in this field.
+
+Advanced
+--------
+
+Your sensor can have advanced options configured per SSID. Select the SSID from the dropdown menu to configure advanced options.
+
+Configure IP:
+
+You can configure your sensor to use DHCP or static IP addresses. For static IP addresses, the following options are available:
```
Comment (rsdouglas) [inline docs/source/dashboard/configuration/sensors_page.rst]: >or a static IP address

(given that this is already per ssid)

File: docs/source/dashboard/configuration/sensors_page.rst [hunk 4]
```diff
@@ -11,11 +11,58 @@ The sensors tab consists of 2 sections:
...
+
+
+- IP Address*: The IPv4 address of the sensor.
+- Subnet Mask*: The subnetwork mask of the IP address.
+- Gateway*: The gateway or router that the sensor will use.
+- Primary DNS Server*: The IPv4 address of the primary DNS server.
+- Secondary DNS: The IPv4 address of the secondary DNS Server.
+- Search Domain: The fully qualified domain name (FQDN) used by the resolver to resolve a FQDN from a relative name.
```
Comment (rsdouglas) [inline docs/source/dashboard/configuration/sensors_page.rst]: Is this optional?

**Comment by qstorey** [comment]
I forgot about this PR 🍰

**Comment by mchampanis** [comment]
how do you still have access 👀

**Comment by iamkhaya** [comment]
i also still have access

**Comment by mchampanis** [comment]
oh, public repo. ya'll gave me a heart attack

**Comment by akiarie** [comment]
hi everyone

**Pull request closed without merging**